client/components/jetpack/scan-threats/index.tsx

```diff
@@ -174,9 +174,8 @@ const ScanThreats = ( { error, site, threats }: Props ) => {
 						count: threats.length,
 					}
 				) }
-				<br />
 				{ translate(
-					'Please review them below and take action. If you have any questions, we are {{a}}here to help{{/a}}.',
+					'Please review and take action. If you have any questions, we are {{a}}here to help{{/a}}.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.